packages/graphql/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hops-graphql",
+  "version": "7.2.0",
+  "description": "React and GraphQL implementation for Hops",
+  "keywords": [
+    "hops",
+    "react",
+    "redux",
+    "graphql"
+  ],
+  "license": "MIT",
+  "bin": {
+    "hops-graphql": "commands/graphql.js"
+  },
+  "main": "node.js",
+  "browser": "dom.js",
+  "files": [
+    "node.js",
+    "dom.js"
+  ],
+  "dependencies": {
+    "apollo-cache-inmemory": "^0.2.0-rc.4",
+    "apollo-client": "^2.0.0-rc.7",
+    "apollo-link-http": "^1.0.0",
+    "graphql": "^0.11.7",
+    "hops-redux": "7.2.0",
+    "isomorphic-fetch": "^2.2.1"
+  },
+  "peerDependencies": {
+    "graphql-tag": "^2.5.0",
+    "react-apollo": "^2.0.0-rc.0"
```

Comment:
I added `react-apollo` as a peerDependency because it exports the `graphql` HOC people have to import in their projects.

Similarly, `graphql-tag` is optionally being used in `hops-build-config` and `jest-preset-hops`.

Does that make sense to you?